team/tests/org.eclipse.compare.tests/src/org/eclipse/compare/tests/DocLineComparatorTest.java

```diff
@@ -269,55 +269,65 @@ public boolean canCacheFilteredRegions() {
 	}
 
 	@Test
-	public void whiteSpaceInStringLiteralWithEmptyIgnoreWhitespaceContributor() {
+	public void noWhitespaceContributorSupplied_whitespaceInStringLiteralIgnored() {
 		DocLineComparator left = new DocLineComparator(new Document("str = \"Hello World\""), null, true, null, 'L', Optional.empty());
 		DocLineComparator right = new DocLineComparator(new Document("str = \"HelloWorld\""), null, true, null, 'R', Optional.empty());
-		Assert.assertTrue(left.rangesEqual(0, right, 0));
-		Assert.assertTrue(right.rangesEqual(0, left, 0));
+		Assert.assertTrue("whitespace in left document between 'Hello' and 'World' not ignored",left.rangesEqual(0, right, 0));
+		Assert.assertTrue("whitespace in left document between 'Hello' and 'World' not ignored",right.rangesEqual(0, left, 0));
```

Comment:
```suggestion
		Assert.assertTrue("whitespace in right document between 'Hello' and 'World' not ignored",right.rangesEqual(0, left, 0));
```

Reply:
> This will more irritate, I think. If you see this error message, you look into the code and see two variables left and right. The whitespace is only on the left DocLineComparator instance; not on the right.

ok got it.